docs/graph.md

```diff
@@ -190,6 +190,17 @@ stateDiagram-v2
   Increment --> DivisibleBy5
 ```
 
+!!! tips
```

Comment:
`tips` seems weird. I'm not sure this needs to be an admonition, but if it is, it needs a better title.